docs/reference/docs/update.asciidoc

```diff
@@ -200,6 +200,8 @@ including:
 `routing`::     Sets the routing that will be used to route the 
                 document to the relevant shard.
 
+`parent`::      Simply sets the routing.
```

Comment:
Maybe give more details here to say that it cannot be used to update the parent, but to locate the document and also when using upsert?